api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPatientDaoImpl.java

```diff
@@ -85,9 +86,44 @@ public PatientIdentifierType getPatientIdentifierTypeByNameOrUuid(String name, S
 			criteria.createAlias("addresses", "pad");
 			criteria.add(c);
 		});
+		if (sort != null) {
+			String paramName = sort.getParamName();
+			if ((paramName.equals("name") || paramName.equals("given") || paramName.equals("family"))
```

Comment:
@CaptainDaVinci  I would suggest you use `Patient.SP_NAME` instead of name `Patient.SP_GIVEN` instead of given `Patient.SP_FAMILY` instead of family

Reply:
@VaishSiddharth I don't think these changes are feasible because there's already an "import org.openmrs.Patient;" line. Otherwise, it'd have to always written like "org.hl7.fhir.r4.model.Patient.SP_NAME".

Reply:
I am not very sure about what you are saying but please have a look at [this](https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/test/java/org/openmrs/module/fhir2/api/dao/impl/FhirEncounterDaoImplTest.java#L134) maybe it'll help.

Reply:
No, I am not saying that it can't be done. All I am saying is that in PatientDaoImpl, patient cannot be imported from two different locations.

Reply:
Okay I understood what your are trying to say. Not sure that which approach is better I think for code cleanliness we should use `name` etc but I guess using `org.hl7.fhir.r4.model.Patient.SP_NAME` will be more fault tolerant.
@ibacher What is your take on this?

Reply:
I too prefer using named constants instead of literals. 

One way to go about having a less verbose approach would be to create an alias,
```
class FHIRPerson extends org.hl7.fhir.r4.model.Patient {
			
}
...
 case FHIRPatient.SP_NAME:
...
```

Reply:
Frankly, I wouldn't get too hung-up on the constants vs literals thing. Either are fine. I certainly wouldn't support introducing a new class just to have the name of the constants to be shorter. In that case, it might make more sense to just use static imports, so you can just write:

```java
...
case SP_NAME:
...
```